### Монету подбросили 144 раза. Какова вероятность, что орел выпадет ровно 70 раз?

Используем биномиальное распределение, так как:

вероятность выпадения орла - известна и она достаточно высока (50%)

количество опытов не слишком высоко

Формула биномиального распределения: P(n)(X=k) = C(n,k) * p^k * q^(n-k)

В данной задаче:

n = 144 - общее количество бросков

k = 70 - выпадения орла

p = 50% = 0.5 - вероятность выпадения орла

q = 1 - 0.5 = 0.5 - обратная вероятность (вероятность выпадения решки)

Найдем сочетания выборки k из n опытов

In [1]:
from math import factorial

c = factorial(144) / (factorial(70) * factorial(144 - 70))
c

1.4007495090837087e+42

2. Найдем p^k и q^(n-k)

In [2]:
p = 0.5 ** 70
q = 0.5 ** (144-70)
p, q

(8.470329472543003e-22, 5.293955920339377e-23)

3. Найдем вероятность того, что выпадет 70 раз орел из 144 бросков

In [3]:
c * p * q

0.06281178035144776

Ответ: Вероятность выпадения орла 70 раз из 144 бросков монетки = 6.28%